AlfrescoApp/Constants/Constants.h

```diff
@@ -272,17 +272,28 @@ extern NSString * const kAlfrescoMainMenuConfigurationViewTypePeople;
 extern NSString * const kAlfrescoMainMenuConfigurationViewTypeGallery;
 extern NSString * const kAlfrescoMainMenuConfigurationViewTypeNodeDetails;
 extern NSString * const kAlfrescoMainMenuConfigurationViewTypeDocumentDetails;
-extern NSString * const kAlfrescoMainMenuConfigurationViewTypeRepositorySearch;
 extern NSString * const kAlfrescoMainMenuConfigurationViewTypeSite;
+extern NSString * const kAlfrescoMainMenuConfigurationViewTypeRepositorySearch;
```

Comment:
Since "Search" is used first in two other constants, can we change this to `kAlfrescoMainMenuConfigurationViewTypeSearchRepository`